### PR TITLE
MOSIP-33679 Inji verify - Automation scripts for UI test cases

### PR DESCRIPTION
**Pull request by anup-nehe**
Inji verify - Automation scripts for UI test cases

### CODE REVIEW

**Review by lsivanand — APPROVED** [review]

File: inji-verify-test/src/test/java/utils/BaseTest.java [hunk 1]
```diff
@@ -30,13 +24,16 @@ public BaseTest(DriverManager driver) {
...
+        Browser.NewContextOptions contextOptions = new Browser.NewContextOptions()
+                .setPermissions(java.util.List.of("camera"));
+        BrowserContext context = browser.newContext(contextOptions);
+        
+		Page page = context.newPage();
 		driver.setPage(page);
-		JSONObject config = baseTestUtil.readConfig(BaseTest.class, ENVIRONMENT);
-		String url = config.getString("url");
-		driver.getPage().navigate(url);
+		driver.getPage().navigate("https://injiverify.qa-inji.mosip.net/");
```
Comment (mohanachandran-s) [inline inji-verify-test/src/test/java/utils/BaseTest.java:34]: Reading the URL from config file is getting overridden. Please correct it.

File: inji-verify-test/src/test/java/utils/BaseTest.java [hunk 2]
```diff
@@ -1,25 +1,19 @@
...
+public class BaseTest{
+	
 	public static Playwright playwright;
 	public static Browser browser;
-
-	protected static final String ENVIRONMENT = System.getProperty("env") == null ? "qa-inji"
-			: System.getProperty("env");
-
-	BaseTestUtil baseTestUtil = new BaseTestUtil();
-
```
Comment (mohanachandran-s) [inline inji-verify-test/src/test/java/utils/BaseTest.java:22]: Don't delete these lines as these are required to read the url's from config file.

File: inji-verify-test/src/test/java/utils/BaseTestUtil.java
Comment (mohanachandran-s) [inline inji-verify-test/src/test/java/utils/BaseTestUtil.java:1]: Don't delete this file

File: inji-verify-test/src/test/resources/config.json
Comment (mohanachandran-s) [inline inji-verify-test/src/test/resources/config.json:1]: Don't delete this file as well

File: inji-verify-test/src/test/resources/featurefiles/UploadQRCodepage.feature [hunk 3]
```diff
@@ -0,0 +1,78 @@
+Feature: Inji Verify upload qr code testing
+
+  @smoke @verifyuploadqrcode
+  Scenario: Verify upload qr code
+
+    Given User gets the title of the page
+    Then Validate the title of the page
+		And Verify that upload QR Code tab is visible
```
Comment (mohanachandran-s) [inline inji-verify-test/src/test/resources/featurefiles/UploadQRCodepage.feature:8]: Please correct the indentation

File: inji-verify-test/src/test/resources/featurefiles/scanQRcode.feature [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    And Verify scan qr code step3 description
+    And verify scan qr code area
+    And verify scan qr code icon
+    And verify scan qr code button
+    And verify click on scan qr code button    
+    And Verify scan qr code step2 label after
+    And verify click on back button
+    And verify scan qr code button
+    And Click on Home button
+		And Verify that Upload button visible
```
Comment (mohanachandran-s) [inline inji-verify-test/src/test/resources/featurefiles/scanQRcode.feature:25]: Correct the indentation

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

**Review by mohanachandran-s — APPROVED** [review]

**Review by challabeehyv — APPROVED** [review]

**Pull request merged**